### ISSUE TITLE
nbgallery thumbnails fill width with sphinx-gallery 0.11

### ISSUE THREAD

**Issue by hakonanes** [spatialaudio/nbsphinx#655]
This `nbsphinx` gallery:

```rst
Reference frames
================

These tutorials cover the relevant reference frames in EBSD and how to determine the
detector-sample geometry, also known as the projection/pattern center (PC).

.. nbgallery::

    reference_frames
    pc_calibration_moving_screen_technique
```

looks like this with sphinx-gallery 0.10.1:

![image](https://user-images.githubusercontent.com/12139781/181599916-6ceddd57-e2cb-4feb-a4c6-98af6702cd76.png)

and like this with sphinx-gallery 0.11:

![image](https://user-images.githubusercontent.com/12139781/181600019-333611cc-0934-4294-af29-4ba6d2067b86.png)

I'm afraid I don't know the reason for this. Are anyone else seeing this as well?

As a temporary solution, I'm pinning `sphinx-gallery to < 0.11` in my projects.

**Comment by kburns** [comment]
Yes seeing the same here, also pinning sphinx-gallery < 0.11 for now.

**Comment by mgeier** [comment]
Thanks for reporting this!

I can reproduce the problem, but I will only have time to look into this after next week.

If somebody has any additional information, please let me know!

I'm of course also open for a pull request!

**Comment by hakonanes** [comment]
> I will only have time to look into this after next week.

That would be great! However, there is no rush to amend this behaviour with sphinx-gallery v0.11 from my side, at least.

**Comment by mgeier** [comment]
I've made a first attempt for solving this at #666.

**Comment by mgeier** [comment]
I'm leaning towards creating custom HTML and CSS for `nbsphinx` and to stop being compatible with Sphinx Gallery (https://github.com/sphinx-gallery/sphinx-gallery/pull/1005#issuecomment-1289433992).

See https://github.com/mgeier/nbsphinx/pull/2 for a discussion, feel free to chime in!

**Comment by mgeier** [comment]
I'm working on a potential solution in #706, please review!

**Comment by hakonanes** [comment]
I built my documentation locally with the changes in #706, and found the changes to solve my issue.

The gallery tiles look very nice. And I like that the tooltip shows the full notebook description. Thank you!
